custom_components/midea_ac_lan/fan.py

```diff
@@ -184,7 +184,10 @@ def __init__(self, device: MideaACDevice, entity_key: str) -> None:
         """Midea AC Fresh Air Fan entity init."""
         super().__init__(device, entity_key)
         self._attr_supported_features = (
-            FanEntityFeature.SET_SPEED | FanEntityFeature.PRESET_MODE
+            FanEntityFeature.SET_SPEED
+            | FanEntityFeature.PRESET_MODE
+            | FanEntityFeature.TURN_ON
+            | FanEntityFeature.TURN_OFF
```

Comment:
`TURN_ON` and `TURN_OFF` was introduced in 2024.8, this will break compatibility with older versions.

Please consider adding a version check.
and set `_enable_turn_on_off_backwards_compatibility=False`  [see](https://developers.home-assistant.io/blog/2024/07/19/fan-fanentityfeatures-turn-on_off)